seed/retrieveImgs/libs.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+
+const {
+  IMG_SIZE,
+  THUMBNAIL_SIZE,
+  IMG_PATH,
+  PRODUCT_IMAGE_COUNT_RANGE,
+} = require('./config');
+
+
+const saveImgToFile = (imageId, name, isThumbnail = false) => new Promise((resolve, reject) => {
+  let size;
+  let thumbnailPostfix = '';
+
+  if (isThumbnail) {
+    size = THUMBNAIL_SIZE;
+    thumbnailPostfix = '-thumbnail';
+  } else {
+    size = IMG_SIZE;
+  }
+
+  const imageUrl = `https://i.picsum.photos/id/${imageId}/${size.join('/')}.jpg`;
+
+  const filename = `${name}${thumbnailPostfix}.jpg`;
+
+  const filePath = path.resolve(IMG_PATH, filename);
+
+
+  axios(imageUrl, {
+    responseType: 'arraybuffer',
+    timeout: 20000,
+  })
+    .then((response) => Buffer.from(response.data, 'binary'))
+    .then((imageBuffer) => {
+      fs.writeFile(filePath, imageBuffer, (err) => {
+        if (err) {
+          console.log(err);
+        } else {
+          console.log('✔️  ', filePath);
```

Comment:
found another one. I like the symbol--gives a nice visual when things are working as you are developing the code.